web/packages/teleport/src/Discover/Server/DiscoveryConfigSsm/DiscoveryConfigSsm.tsx

```diff
@@ -62,6 +64,25 @@ import { DiscoveryConfigCreatedDialog } from './DiscoveryConfigCreatedDialog';
 
 const IAM_POLICY_NAME = 'EC2DiscoverWithSSM';
 
+type AWSLabel = {
+  name: string;
+  value: string;
+};
+
+type AWSLabels = AWSLabel[];
+
+function makeTags(labels: AWSLabels): Labels {
+  if (labels.length === 0) {
+    return { '*': ['*'] };
+  }
+  const output: Labels = {};
+
+  labels.forEach(label => {
+    output[label.name] = [label.value];
```

Comment:
does this support multi value array? (it just looks like we are replacing the array instead of appending)

Reply:
no multivalue